pkg/assets/handlers.go

```diff
@@ -210,6 +212,13 @@ type WebConsoleConfig struct {
 	LoggingURL string
 	// MetricsURL is the endpoint for metrics (optional)
 	MetricsURL string
+	// LimitRequestOverrides contains the ratios for overriding request/limit on containers.
+	// Applied in order:
+	//   Enabled (no overrides apply if not enabled)
+	//   LimitCPUToMemoryRatio
+	//   CPURequestToLimitRatio
+	//   MemoryRequestToLimitRatio
+	LimitRequestOverrides api.PodLimitRequestConfig
```

Comment:
I think it's causing the problem @spadgett pointed out above. You likely need to put the individual fields here.